components/clm/src/data_types/ColumnDataType.F90

```diff
@@ -107,6 +107,7 @@ module ColumnDataType
     real(r8), pointer :: h2osoi_ice         (:,:) => null() ! ice lens (-nlevsno+1:nlevgrnd) (kg/m2)    
     real(r8), pointer :: h2osoi_vol         (:,:) => null() ! volumetric soil water (0<=h2osoi_vol<=watsat) (1:nlevgrnd) (m3/m3)  
     real(r8), pointer :: h2osfc             (:)   => null() ! surface water (kg/m2)
+    real(r8), pointer :: salinity           (:)   => null() ! salinity from PFLOTRAN when using interface (TAO 5/19/2020)
```

Comment:
Just a placeholder for my work, does not need to be incorporated in current code

Reply:
Sophie and I were talking about integrating salinity for her updates so I think we'll keep this, unless it makes sense to put it somewhere else

Reply:
My only concern is that it may need to be called something else since salinity is already in E3SM in the ocean module -  not sure if that actually makes a difference since we're not interacting with that one, but maybe something to consider when its time to run coastal as part of the global model